README.md

```diff
@@ -327,11 +327,11 @@ Body:
 
 ##### 3.3.2.2.2. CDE Upload Response
 
-The server returns a response to inform the client about the `upload_ui_url` which should be opened in a local web browser. The `expires_in` property specifies how long this link will be valid, and the optional property `max_size_in_bytes` can be used to indicate if there is a file size limit for uploading.
+The server returns a response to inform the client about the `upload_ui_url` which should be opened in a local web browser. The `expires_in` property specifies the URL's validity, in seconds, and the optional property `max_size_in_bytes` can be used to indicate if there is a file size limit for uploading.
```

Comment:
I'm not sure, but should it be here `the URLs validity` without the `'`?

Reply:
I think its fine Georg - its "the validity of the URL" and I believe the apostrophe should be there